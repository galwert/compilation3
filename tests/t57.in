int getInput() {
    return 1; // TODO - Actually read input from user
}

void main(){
    int x = getInput();
    int y = getInput();
    int XplusY = x + y;
    int XdivY = getInput() / y;
    bool XeqY = x == y;
    bool XgtY = x > y;
    bool XltY = x < y;
    bool xneY = x != printi(7); // error mismatch

    if (x != y) {
        return;
    }

    x = getInput();
    y = getInput();

    XgtY = x >= y;
    XltY = x <= y;
    XgtY = x > y;
    XltY = x < y;

    XneY = not XeqY;
    XeqY = (not XgtY) and (not XltY);
    if (((not XgtY) and (not XltY)) or (not (not (not XeqY))))


}