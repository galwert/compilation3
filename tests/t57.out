---end scope---
line 13: type mismatch
